src/routes/Profile/index.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable sonarjs/no-duplicate-string */
```

Comment:
We should maybe avoid such rules.

Reply:
Yeah, or moving the values to constants to keep it happy. Separate PR to handle these duplicate rules one way or another